src/SFA.DAS.ApplyService.Database/Tables/OrganisationAddresses.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE TABLE [dbo].[OrganisationAddresses]
+(
+	[Id] INT NOT NULL IDENTITY(1,1),
+	[OrganisationId] UNIQUEIDENTIFIER NOT NULL,
+	[AddressType] [TINYINT] NOT NULL,
+	[AddressLine1] NVARCHAR(MAX) NULL,
+	[AddressLine2] NVARCHAR(MAX) NULL,
+	[AddressLine3] NVARCHAR(MAX) NULL,
```

Comment:
Do AddressLine1,2,3 needs to be Max size ?

Reply:
Address is not user input, we derive it from external API so it really does not matter.